imports/plugins/core/catalog/server/no-meteor/mutations/publishAllProducts.js

```diff
@@ -0,0 +1,10 @@
+import publishProducts from "./publishProducts";
+
+
+export default async function publishAllProducts(context) {
+  const { collections } = context;
+  const { Products } = collections;
+  const allProducts = await Products.find({ type: "simple" }, { _id: 1 }).toArray();
```

Comment:
Does this fetch the entire product collection to the server and then work with the entire set in memory?